indico/modules/rb_new/client/js/common/rooms/RoomEditModal.module.scss

```diff
@@ -27,3 +27,8 @@
     margin-bottom: 10px;
 }
 
+.room-form {
+    :global(.fields) > :global(.field) {
+        flex: 1 1 auto !important;
```

Comment:
I was going to suggest to not use the shorthand property, but at least https://css-tricks.com/snippets/css/a-guide-to-flexbox/ recommends it.. :/

Anyway, the second/third params are optional so unless you need them to override something let's omit them.

Reply:
This is a fix to how (as far as I know) Chrome renders flex elements with `flex-basis: auto`. On the other hand I think it will be enough to just override `flex-grow` here because that is the only thing I care about.